src/pages/SingleVideoPage.vue

```diff
@@ -1,96 +1,181 @@
-                <rect x="0" y="0" width="4" height="4" class="text-gray-200" fill="currentColor" />
+              <pattern
+                id="02f20b47-fd69-4224-a62a-4c9de5c763f7"
+                x="0"
+                y="0"
+                width="20"
+                height="20"
+                patternUnits="userSpaceOnUse"
+              >
+                <rect
+                  x="0"
+                  y="0"
+                  width="4"
+                  height="4"
+                  class="text-gray-200"
+                  fill="currentColor"
+                />
               </pattern>
             </defs>
-            <rect width="404" height="392" fill="url(#02f20b47-fd69-4224-a62a-4c9de5c763f7)" />
+            <rect
+              width="404"
+              height="392"
+              fill="url(#02f20b47-fd69-4224-a62a-4c9de5c763f7)"
+            />
           </svg>
         </div>
-        <div class="relative mx-auto max-w-md px-4 sm:max-w-3xl sm:px-6 lg:px-0 lg:max-w-none lg:py-20">
+        <div
+          class="
+            relative
+            mx-auto
+            max-w-md
+            px-4
+            sm:max-w-3xl
+            sm:px-6
+            lg:px-0
+            lg:max-w-none
+            lg:py-20
+          "
+        >
           <SingleVideo v-if="authorStatus === 'ready'" :id="embed(content)" />
         </div>
       </div>
 
       <div class="relative mx-auto max-w-md px-4 sm:max-w-3xl sm:px-6 lg:px-0">
         <!-- Content area -->
         <div class="pt-12 sm:pt-16 lg:pt-20">
-          <SingleVideoTitle v-if="authorStatus === 'ready'" :video="content" />
-          <!--
-          <div class="mt-10 border-t-2 border-gray pt-6">
-            <a href="#" target="_blank" class="text-base font-medium">
-              {{ t('videos.learnMore') }}
-              <span v-if="contentStatus === 'ready'" aria-hidden="true">{{ content.title }} &rarr;</span>
-            </a>
-          </div>
-          -->
+          <SingleVideoTitle :video="content" />
         </div>
       </div>
     </div>
   </div>
+  <div v-if="authorStatus === 'notAvailable'" class="text-center pt-16 sm:pt-24">
+    <h1>Sorry, this video is not available in the selected language.</h1>
+    <h2 class="pb-8">
+      Would you like to contribute and make a related video in your
+      language?
+    </h2>
+    <ContributionCTA />
+  </div>
 </template>
 <script>
 /* eslint-disable */
-import { onMounted, watch, toRaw } from 'vue';
+import { onMounted, watch, toRaw } from "vue";
 
-import { useI18n } from 'vue-i18n';
+import { useI18n } from "vue-i18n";
 
-import SingleVideo from '@/components/SingleVideo';
-import SingleVideoTitle from '@/components/SingleVideoTitle';
+import SingleVideo from "@/components/SingleVideo";
+import SingleVideoTitle from "@/components/SingleVideoTitle";
+import ContributionCTA from "@/components/ContributionCTA";
```

Comment:
again it's a prettier issue, it changes all single quotes to double quotes. can you please revert this back to single quotes? check this out https://stackoverflow.com/questions/48864985/vscode-single-to-double-quote-automatic-replace

Reply:
My editor is automatically changing these. So that's why I created #72 #73 hopefully we can get rid of this different styling. we have different styles everywhere.